### PR TITLE
128 l container calibration component

### PR DESCRIPTION
**Pull request by Kadee80**
This PR:

- creates a CalibratePlaceable component
- save button posts to backend route /calibrate placeable
- returns updated steplist to update task store (calibrated = true)
- when calibrated = true for placables, completed class applied to calibrate step in steplist component
- enables move to button for calibrated placeables
- merges in from master several PRS from master to avoid vuex refactor conflicts and allow for calibration via API and backend routes

### CODE REVIEW

File: app/renderer/src/store/actions.js [hunk 1]
```diff
@@ -29,12 +29,7 @@ const actions = {
     OpenTrons.uploadProtocol(formData).then((result) =>{
       if (result.success) {
         let tasks = result.calibrations
-        tasks.map((instrument) => {
-          instrument.href = instrumentHref(instrument)
-          instrument.placeables.map((placeable) => {
-            placeable.href = placeableHref(placeable, instrument)
-          })
-        })
+        let _tasks = mapRoutes(tasks)
```
Comment (levthedev) [inline app/renderer/src/store/actions.js]: are you ever using this _tasks variable?

Comment (Kadee80) [inline app/renderer/src/store/actions.js]: i am, _tasks is whats returned when you map over the returned tasks from the backend

Comment (Kadee80) [inline app/renderer/src/store/actions.js]: its mutating the returned from the backend - not from the existing tasks in store

**Comment by levthedev** [comment]
This can be merged once the tests are fixed

**Pull request merged**